addons/binding/org.openhab.binding.atlona/src/main/java/org/openhab/binding/atlona/internal/net/SocketChannelSession.java

```diff
@@ -1,4 +1,4 @@
-package org.openhab.binding.russound.internal.net;
+package org.openhab.binding.atlona.internal.net;
```

Comment:
Bit confused - I copied these files from the russound binding and accidentally committed them without changing the namespace on them.  I then updated them to the altona namespace and recommitted (probably should have merged commits as well).  Unless I'm misunderstanding you - it's correct the way it is.